src/lib/containers/GenericCard.tsx

```diff
@@ -70,8 +72,24 @@ export default class GenericCard extends React.Component<GenericCardProps, Gener
     } else if (link.match(DOI_REGEX)) {
       target = '_blank'
       linkDisplay = `https://dx.doi.org/${link}`
-    } else if (!hasInternalLink) {
+    } else if (!internalLinkConfiguration) {
       target = '_blank'
+    } else if (internalLinkConfiguration) {
+      if (!data || !schema) {
+        throw Error('Must specify internalLinkConfiguration and data for linking to work')
+      }
+      const columnValuesLength = internalLinkConfiguration.columnValues.length
+      const urlParams = internalLinkConfiguration.columnValues.map(
+        (el, index) => {
+          if (!schema.hasOwnProperty(el)) {
+            console.error(`Could not find match for data: ${data} with columnName ${el}`)
+          }
+          const stringEnd = index < columnValuesLength - 1 ? '&' : ''
+          return `${el}=${data[schema[el]]}${stringEnd}`
+        }
+      ).join('')
+      // tested this link on the browser, there's no need to encode the URL, the browser picks up on that automatically
```

Comment:
did you have a chance to verify on Chrome + FF + Safari?  should be ok, but just want to make sure!